charts/pgadmin4/values.yaml

```diff
@@ -130,10 +138,11 @@ existingSecret: ""
 
 ## pgAdmin4 startup configuration
 ## Values in here get injected as environment variables
-##
+## Needed chart reinstall for apply changes
 env:
-  email: chart@example.local
+  email: chart@example.local  # can be email or nickname
```

Comment:
Could you place this comment above rather then behind?

Reply:
Sure, fixed